mlir_graphblas/functions.py

```diff
@@ -36,10 +36,43 @@ def __init__(self):
     def get_mlir(self, *, make_private=True):
         """
         `make_private` is used to indicate whether the function should be private for fusion
-        or public for standalone calling in the `compile` method
+        or public for standalone calling in the `compile` method.
+
+        `module_wrapper` indicates whether to wrap the function text in a MLIR module
```

Comment:
This docstring update seems out of date. I assume you started with this approach, then moved to a separate `get_mlir_module` function.